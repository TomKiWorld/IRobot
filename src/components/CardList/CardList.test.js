import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

describe('CardList Component', () => {
  it('Expect to render CardList Component', () => {
    const mockRobots = [
      {
        id: 1,
        name: 'John Snow',
        email: 'john@gmail.com'
      }
    ]
    expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
  });
});
