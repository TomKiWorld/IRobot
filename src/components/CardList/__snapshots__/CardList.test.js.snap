// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList Component Expect to render CardList Component 1`] = `ShallowWrapper {}`;
